build.sbt

```diff
@@ -111,6 +113,43 @@ lazy val `cluster-bootstrap` = project
   )
   .dependsOn(`akka-management`)
 
+lazy val `lease-kubernetes` = project
+  .in(file("lease-kubernetes"))
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "akka-lease-kubernetes",
```

Comment:
organization is by default `organization := "com.lightbend.akka.management"`, I guess that works for this

Reply:
bit weird to have management in the name but would also be weird to have a project in this repo without management in the name

Reply:
On the other hand akka-coordination is in com.typesafe.akka and we have used com.lightbend.akka or com.typesafe.akka also for things located outside the akka/akka repo.

Not sure, but we could use `com.lightbend.akka`.

Reply:
bootstrap is using `com.lightbend.akka.management`. Maybe we'll just stick to that instead of adding more?

Reply:
Ah sorry it was the discovery impls that didn't use management in the name e.g. `com.lightbend.akka.discovery`

Yes let's just stick to com.lightbend.akka.management and be done with it!